src/minerva_lib/blend.py

```diff
@@ -1,59 +1,58 @@
 import numpy as np
+import skimage.exposure
 
 
-def to_f32(img):
-    '''Scale the dynamic range to 0.0 - 1.0
+def composite_channel(a, image, color, range_min, range_max, out=None):
+    ''' Composite given image _a_ with rendered channel _image_
 
-    Arguments:
-    img: An integer image
-    '''
+    To update _a_ destructively, pass the same array to _a_ and _out_.
```

Comment:
Change to`By default, a new output array will be allocated to hold the result of the composition operation. To update _target_ in place instead, specify the same array for _target_ and _out_.`